mlir/lib/Conversion/Normalize/Normalize.cpp

```diff
@@ -0,0 +1,436 @@
+//===- Normalize.cpp - Conversion from MLIR to its canonical form ---------===//
```

Comment:
Why not make this pass local to a tool?

Reply:
Yes, although I notice that LLVM's is a regular pass too. So fine to keep as regular pass.